### PR TITLE
apply message styles

### PR DESCRIPTION
**Pull request by giannik**
apply bootstrap styles to message based on notification type

### CODE REVIEW

File: src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="alert alert-warning alert-dismissible fade show" role="alert">
+<div class="alert @(Model.Type.ToString() == "Error" ? "alert-danger" : Model.Type.ToString() == "Success"? "alert-success": Model.Type.ToString() == "Information" ? "alert-info" : "alert-warning") alert-dismissible fade show" role="alert">
```
Comment (hishamco) [inline src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: Could we use the pattern matching?

Comment (giannik) [reply src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: an example ?

**Comment by deanmarcussen** [comment]
This shape was done incorrectly in the first instance.

It needs to use the `TagBuilder` pattern, then it would take the correct classes (for which we have the correct css already) from the Core `Message` shape, and the `alert-dismissable` etc classes could be added to the existing `Message` shape. 

@hishamco I am sure I explained this on the original pr, can you find the example from there for @giannik please

**Comment by hishamco** [comment]
Ya, I remember this one, let me shoot the original PR that I made

**Comment by hishamco** [comment]
@deanmarcussen I think you mean this one https://github.com/OrchardCMS/OrchardCore/pull/4654

**Comment by deanmarcussen** [comment]
@giannik I have updated this to use the existing css styles we have for the message-type (they just extend the bootstrap ones).

If you agree with the change I will approve the pr

File: src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml [hunk 2]
```diff
@@ -1,4 +1,7 @@
-<div class="alert alert-warning alert-dismissible fade show" role="alert">
+@{
+    string type = Model.Type.ToString().ToLowerInvariant();
+}
+<div class="alert alert-dismissible fade show message-@type" role="alert">
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: Why not using `Model.Type.ToString().ToLowerInvariant();` directly instead?

**Comment by giannik** [comment]
> @giannik I have updated this to use the existing css styles we have for the message-type (they just extend the bootstrap ones).
> 
> If you agree with the change I will approve the pr

sure @deanmarcussen fine by me.

File: src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml (hunk 2 above)
Comment (deanmarcussen) [reply src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: Yes probably could be @Skrypt I did it with `string type` as it is dynamic, and a bit more explicit. will check tomorrow

Comment (Skrypt) [reply src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: Well I tried with my website and it works fine. You cast it as a String when you do .ToString() anyways 😄

**Pull request merged**